presto-spark-launcher/src/main/java/com/facebook/presto/spark/launcher/PrestoSparkClientOptions.java

```diff
@@ -34,4 +34,7 @@
 
     @Option(name = "--schema", title = "schema", description = "Default schema")
     public String schema;
+
+    @Option(name = "--session-property", title = "session-property", description = "session-property-configuration.properties path")
```

Comment:
According to `file`, `packagePath`, `config`, shouldn't this be:
name = "--session-property-config", title = "file" ....